### DISCUSSION TITLE
[Suggestion] Bring back the heat bar

### DISCUSSION THREAD

**Discussion by PFSCommissar** [tsunamayo/Starship-EVO#4581] [Suggestions]
Although heat death has been removed, being able to monitor the heat is very important for combat, as well as properly engineering a ship to handle the heat loads that it will undergo. Therefore, please bring back the heat bar to the HUD.

**Comment by ProPeach** [comment]
Agreed! Even a smaller percentage heat icon would work great if you don't want it to take up as much GUI space as a bar

**Comment by tsunamayo** [comment]
@PFSCommissar Hi, yes like I will put some sort of an indication when it say > 0.75%, I just dont want to overload the player with info. We might have other status affliction too from weapon, so I need something more low-key. Thanks

**Reply by ProPeach** [reply]
I can definitely empathise the want for simplicity @tsunamayo, there are lots of things going on at once in a space game. However, thought smart GUI design you can include lots of valuable information on screen but only draw the players attention to it when they need to take notice - The player often ignores information on screen when they don't need to see it so you have some wiggle room with putting many bits of information on screen at once.

So you could have a small heat percentage figure up in the GUI permanently to provide instant feedback (important for seeing the rate at which heat changes), but then have it flash red when it goes over 75%. for example.


I think we need this kind of feedback on game systems that are integral to how ships work, like heat. It doesn't make a lot of sense to have heat be such a big part of ship building and combat and then hide it for 75% of the time

**Reply by PFSCommissar** [reply]
Agree with this. The most fun aspect of this game is design and engineering. I’d argue it’s the main purpose of the game. Therefore, whatever facilitates that should take priority, and heat is a key performance parameter of ships. We have to be able to easily measure how quickly heat builds and build around that.

**Reply by Pharap** [reply]
> I just dont want to overload the player with info.

Perhaps it could be toggled with a keybind or setting?
(Which could perhaps be a step towards a customisable ship UI?)

**Comment by tsunamayo** [comment]
sorry I wanted to move this to suggestions...

**Comment by Regnion** [comment]
A heat meter isn't information overload. It would be infuriating if, as a player experiencing overheat, you have no idea what is going on because nothing is informing you. It is also critically important for the ship builders, regardless of skill, to know what sort of heat challenge they're facing before they produce a poorly functioning ship. We know our exact thrust output, reactor output, fuel capacity, why is heat something that needs to be nebulous and undefined?

It is worse to deprive us of information, or provide us too little information, than simply give us the meter back.

**Comment by Uncle-Ulty** [comment]
"Bring back! Bring back! Bring back to where it once belonged!!!"
